### ISSUE TITLE
Possible race in TestFlushesAfterInterval test, update_rollup_test.go

### ISSUE THREAD

**Issue by dansimau** [uber/ringpop-go#60]
While running the swim package test suit the `TestFlushesAfterInterval` test failed with:

```
--- FAIL: TestFlushesAfterInterval (0.00s)
        Error Trace:    update_rollup_test.go:106
    Error:      Should be empty, but was map[one:[{ 0 one 0  {63587583130 67548078 0x7b3660}}] two:[{ 0 two 0  {63587583130 67548078 0x7b3660}}]]
    Messages:   expected buffer to be flushed

--- FAIL: TestRollupTestSuite (0.01s)
```

I haven't seen it before and I wasn't able to reproduce it again.

**Comment by dansimau** [comment]
Still happening.
```
--- FAIL: TestFlushesAfterInterval (0.01s)
	Error Trace:	update_rollup_test.go:106
	Error:		Should be empty, but was map[one:[{ 0 one 0  {63589156005 153196154 0xbd6140}}] two:[{ 0 two 0  {63589156005 153196154 0xbd6140}}]]
	Messages:	expected buffer to be flushed
		
--- FAIL: TestRollupTestSuite (0.01s)
```

**Comment by dansimau** [comment]
Fixed by #92. Closing.
